bot/cogs/antimalware.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from discord import Message, NotFound
+from discord.ext.commands import Bot, Cog
+
+from bot.constants import AntiMalware as AntiMalwareConfig, Channels
+
+log = logging.getLogger(__name__)
+
+
+class AntiMalware(Cog):
+    """Cog providing anti-malware behavior."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.whitelist = tuple(AntiMalwareConfig.whitelist)
```

Comment:
Why is this done? As an alias?

Reply:
Yes, to clean up its use later on - `a.filename.lower().endswith(tuple(AntiMalwareConfig.whitelist))` just looked ugly and unclear to me. Also I think I originally intended to use the whitelist more than once for some reason.

What do you prefer?

Reply:
Since we're suggesting using a normal for loop, I don't see this alias as useful any more.